README.md

```diff
@@ -94,8 +94,9 @@ The options under `hooks` specify the meat of a migration. They tell Shepherd ho
 - `post_checkout` is a sequence of commands to be executed once a repo has been checked out and passed any `should_migrate` checks. This is a convenient place to do anything that will only need to be done once per repo, such as installing any dependencies.
 - `apply` is a sequence of commands that will actually execute the migration. This example is very simple: we're just using `mv` to rename a file. However, this hook could contain arbitrarily many, potentially complex commands, depending on the requirements of your particular migration.
 - `pr_message` is a sequence of commands that will be used to generate a pull request message for a repository. In the simplest case, this can just be a static message, but you could also programmatically generate a message that calls out particular things that might need human attention. Anything written to `stdout` will be used for the message. If multiple commands are specified, the output from each one will be concatenated together.
-
-`should_migrate` and `post_checkout` are optional; `apply` and `pr_message` are required.
+- `issue_message` is a sequence of commands that will be used to post a issue message for a repository. In the simplest case, this can just be a static message.
+- `issue_labels` is a sequence of commands that will be used to add labels to the issue posted to the repository.
+  `should_migrate` and `post_checkout` are optional; `apply` and `pr_message` are required. `issue_message` and `issue_labels` are required for `create-issue` command.
```

Comment:
I'm thinking we can move this into a Schema section of the readme where it can be quickly referenced for required vs optional keys

Reply:
yes I will refine the documentation in the next PR. Would you want to open an issue?

Reply:
The documentation should accompany the changes.  Happy to contribute towards that end.